lisp/ess-inf.el

```diff
@@ -1759,18 +1727,19 @@ input will fail."
   "Send the current line to the inferior ESS process.
 VIS has same meaning as for `ess-eval-region'."
   (interactive "P")
-  (let* ((beg (point-at-bol))
-         (end (point-at-eol))
+  (let* ((beg (line-beginning-position))
+         (end (line-end-position))
```

Comment:
Shall we use `point-at-eol` instead? I find it more consistent with `point`, `point-min`, `point-max`.